index.html

```diff
@@ -166,6 +188,9 @@ <h3 class="heading-blue-background">Variadic functions Projects</h3>
                 </div>
             </div>
         </div>
+        <div class="geometry-5"></div>
+        <div class="geometry-4"></div>
+
     </section>
     <section class="About">
```

Comment:
- [ ] As requested by the previous reviewer, since your media query starts from a min-wifth of 768px, your desktop design should be visible from 768px. Kindly fix the feel of the about section at this screen width.
- [Optional] Also I noticed you have a horizontal scroll on your desktop design. I think this is because you use a lot of static width in your CSS. Kindly consider using a dynamic width (% or vw). This is better for responsive design.
![image](https://user-images.githubusercontent.com/94551484/222868819-0e111ad4-0be0-411b-86d2-a4104453ef86.png)